### PR TITLE
ENH: Adding gradient accumulation flag to trainer

### PR DESCRIPTION
**Pull request by vale-salvatelli**
Adding gradient accumulation flag to enable gradient accumulation in  Pytorch Lightening containers

### CODE REVIEW

File: hi-ml/src/health_ml/deep_learning_config.py [hunk 1]
```diff
@@ -486,6 +486,8 @@ class TrainerParams(param.Parameterized):
                                               "any validation overheads during training time and produce "
                                               "additional time or memory consuming outputs only once after "
                                               "training is finished on the validation set.")
+    pl_accumulate_grad_batches: int = param.Integer(default=1,
+                                                    doc="how many batches to accumulate before gradient update")
```
Comment (ant0nsc) [inline hi-ml/src/health_ml/deep_learning_config.py:490]:
```suggestion
                                                    doc="How many batches to accumulate before doing gradient update")
```

**Review by ant0nsc — APPROVED** [review]
